docs/technical_reference/unit_models/zero_order_unit_models/feed_zo.rst

```diff
@@ -4,6 +4,7 @@ Feed  (ZO)
 Model Type
 ----------
 This unit model is formulated as a **None** model form.
```

Comment:
For the case of FeedZO, we could provide an alternative section title instead of Model Type and also get rid of that first sentence about the model form.